ramiro_serra/Debug and profiling notes.md

```diff
@@ -82,6 +84,30 @@ We can see the program stopped at the checkpoint we set previously.
         at /create_ws/src/create_autonomy/navigation/ca_move_base/src/send_robot_goal.cpp:10
     10      const std::string NODE_NAME = "navigation_goals";
 
+Now if we use `where full`:

```

Comment:
Do you notice something useful there?

Reply:
Maybe that when the variables are optimized out, you can't see their value?